__tests__/hooks/useGroupInfo.test.js

```diff
@@ -614,35 +614,6 @@ describe('useGroupInfo should run correctly', () => {
     expect(mockAddCreate).toHaveBeenCalled();
     expect(moctAddMemeber).toHaveBeenCalled();
   });
-  it('onRemoveUser should run correctly', async () => {
-    const {result} = renderHook(() => useGroupInfo(), {wrapper});
-    await result.current.setSelectedUser({user_id: '123', name: 'agita', user: {name: 'agita'}});
-    await result.current.setNewParticipant([
-      {user_id: '123', name: 'agita', user: {name: 'agita'}},
-      {user_id: '1234', name: 'elon', user: {name: 'elon'}}
-    ]);
-
-    await result.current.onRemoveUser();
-    expect(result.current.openModal).toBeFalsy();
-    expect(result.current.newParticipant).toEqual([
-      {user_id: '1234', name: 'elon', user: {name: 'elon'}}
-    ]);
-    expect(mockRemoveMember).toHaveBeenCalled();
-    expect(mockGroupDispatch).toHaveBeenCalled();
-  });
-
-  it('error onRemoveUser should run correctly', async () => {
-    const {result} = renderHook(() => useGroupInfo(), {wrapper: wrapperError});
-    await result.current.setSelectedUser({user_id: '123', name: 'agita', user: {name: 'agita'}});
-    await result.current.setNewParticipant([
-      {user_id: '123', name: 'agita', user: {name: 'agita'}},
-      {user_id: '1234', name: 'elon', user: {name: 'elon'}}
-    ]);
-    const spyConsole = jest.spyOn(console, 'log');
-    await result.current.onRemoveUser();
-    expect(spyConsole).toHaveBeenCalled();
-  });
-
   it('handleCloseSelectedUser should run correctly', async () => {
     const {result} = renderHook(() => useGroupInfo(), {wrapper});
     act(() => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the function name `moctAddMemeber`. It should be `mockAddMember`.
```diff
- expect(moctAddMemeber).toHaveBeenCalled();
+ expect(mockAddMember).toHaveBeenCalled();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->